frontend/src/components/search/SearchResultsList.tsx

```diff
@@ -18,17 +18,118 @@ const SearchResultsList: React.FC<SearchResultsListProps> = ({
+                        tablet:border-base-lighter
+                      "
+                  >
+                    <span className="usa-tag bg-accent-warm-dark">
+                      <strong className="">Closing:</strong>{" "}
+                      {/* TODO: format date */}
+                      {opportunity.summary.close_date}
+                    </span>
+                  </span>
+                  <span
+                    className="
+                        display-block
+                        tablet:display-inline-block
+                        tablet:border-left-1px
+                        tablet:padding-x-1
+                        tablet:margin-left-neg-1
+                        tablet:margin-right-1
+                        tablet:border-base-lighter
+                      "
+                  >
+                    <strong>Posted:</strong> {opportunity.summary.post_date}
+                    {/* TODO: format date */}
+                  </span>
+                </div>
+                <div className="grid-col tablet:order-3 overflow-hidden">
+                  <span
+                    className="
+                        display-block
+                        tablet:display-inline-block
+                        tablet:border-left-1px
+                        tablet:padding-x-1
+                        tablet:margin-left-neg-1
+                        tablet:margin-right-1
+                        tablet:border-base-lighter
+                      "
+                  >
+                    <strong>Agency:</strong> {opportunity.summary.agency_name}
+                  </span>
+                  <span
+                    className="
+                        display-block
+                        tablet:display-inline-block
+                        tablet:border-left-1px
+                        tablet:padding-x-1
+                        tablet:margin-left-neg-1
+                        tablet:margin-right-1
+                        tablet:border-base-lighter
+                      "
+                  >
+                    <strong>Opportunity Number:</strong>{" "}
+                    {opportunity.opportunity_number}
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div className="desktop:grid-col-auto">
+              <div className="overflow-hidden">
+                {/* TODO: Better way to format as a dollar amounts */}
+                <span className="display-block desktop:text-right">
+                  <strong>Award Ceiling:</strong>{" "}
+                  <span className="desktop:display-block desktop:font-sans-lg text-ls-neg-3">
+                    ${opportunity.summary.award_ceiling.toLocaleString()}
+                  </span>
+                </span>
+                <span className="display-block desktop:text-right">
+                  <strong>Floor:</strong> $
+                  {opportunity.summary.award_floor.toLocaleString()}
```

Comment:
For the PR check errors you can run `npm run lint && npm run ts:check` from the `./frontend` folder.
I think there's some errors here since values can be null - you can try to put a question mark in those places where it might be null 
`opportunity?.summary?.award_floor.toLocaleString() || ""

It kind of depends on what data we get back so the `searchTypes.ts` file might be wrong also.


Also `npm run format` if there's ever format errors.